docker/env.list

```diff
@@ -29,5 +25,3 @@ KOLIBRI_RUN_MODE="docker"
 # DOCKERMNT_PEX_PATH=
 # Specify path to a .pex file in the /docker/mnt volume, e.g.
 # DOCKERMNT_PEX_PATH=/docker/mnt/kolibri-0.11.0a1.dev_git-37-gb15b923.pex
```

Comment:
There doesn't seem to be a new line at the end of this file, so I wonder if we need to make sure that a new line is added before adding any env vars in the python script above.